### PR TITLE
Stingray Ideas

### PR DESCRIPTION
**Pull request by dpshelio**
@matteobachetti said to bring forward this idea. Are there others to create?

### CODE REVIEW

**Comment by dpshelio** [comment]
To be merged after #268

File: _projects/2021/stingray/multitaper.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+name: Multitaper-Periodograms in Stingray.
```
Comment (mirca) [inline _projects/2021/stingray/multitaper.md:2]: Multi-taper Periodograms?

just to be consistent with the rest of the text.

**Review by mirca — APPROVED** [review]

File: _projects/2021/stingray/multitaper.md (hunk 1 above)
Comment (matteobachetti) [reply _projects/2021/stingray/multitaper.md:2]: There are other places where we used the entire word. Yes, it should be made consistent :)
I don't have editing rights though.

**Comment by matteobachetti** [comment]
@dpshelio we have at least one more, I will open a PR as soon as you merge this ok?

**Comment by dpshelio** [comment]
@matteobachetti merged! Looking forward for the new ideas ;)

**Pull request merged**